www/include/reporting/dashboard/viewHostLog.php

```diff
@@ -58,15 +58,26 @@
 $redirect = $formHost->addElement('hidden', 'o');
 $redirect->setValue($o);
 
-$hosts = getAllHostsForReporting($is_admin, $lcaHoststr);
```

Comment:
If the function (getAllHostsForReporting) is no longer used, remember to delete it.

Reply:
This function is already deleted in the lastest commit

Reply:
Function deleted in db-func.php